packages/pages/src/bundler.js

```diff
@@ -63,11 +63,7 @@ if (pluginFiles.length == 0) {
 }
 mkdirSync(pluginOutputPath, { recursive: true });
 pluginFiles.map((filepath) =>
-  copyFile(
-    filepath,
-    `${pluginOutputPath}${path.basename(filepath)}`,
-    () => undefined
-  )
+  copyFileSync(filepath, `${pluginOutputPath}${path.basename(filepath)}`)
```

Comment:
why change this?

Reply:
I was using copyFileSync in another place so updated the rest. The interface is nicer too since you don't need to handle the errors yourself.